tests/queries/0_stateless/01656_test_query_log_factories_info.sql

```diff
@@ -0,0 +1,58 @@
+SELECT uniqArray([1, 1, 2]),
+       SUBSTRING('Hello, world', 7, 5),
+       flatten([[[BIT_AND(123)]], [[mod(3, 2)], [CAST('1' AS INTEGER)]]]),
+       week(toDate('2000-12-05')),
+       CAST(arrayJoin([NULL, NULL]) AS Nullable(TEXT)),
+       avgOrDefaultIf(number, number % 2),
+       sumOrNull(number),
+       toTypeName(sumOrNull(number)),
+       countIf(toDate('2000-12-05') + number as d,
+       toDayOfYear(d) % 2)
+FROM numbers(100);
+SELECT '';
+
+SYSTEM FLUSH LOGS;
+SELECT arraySort(used_aggregate_functions)
+FROM system.query_log WHERE type = 'QueryFinish' AND (query LIKE '%toDate(\'2000-12-05\')%')
+ORDER BY query_start_time DESC LIMIT 1 FORMAT TabSeparatedWithNames;
```

Comment:
Worth adding `current_database = currentDatabase()` to make it restartable?

Reply:
Yes, let's correct the test...

Reply:
#19675